Services/CO.CDP.EntityVerification/Persistence/DatabasePponRepository.cs

```diff
@@ -0,0 +1,33 @@
+using Microsoft.EntityFrameworkCore;
+
+namespace CO.CDP.EntityVerification.Persistence;
+
+public class DatabasePponRepository(EntityValidationContext context) : IDatabasePponRepository
```

Comment:
I know naming hasn't been consistend in tickets, but it's called "entity verirification" (not "entity validation").

Reply:
Done.